src/build/build_step_test.go

```diff
@@ -507,6 +530,25 @@ func TestSha1SingleHash(t *testing.T) {
 	}
 }
 
+func newStateWithHashCheckers(label, hashFunction string, hashCheckers []string) (*core.BuildState, *core.BuildTarget) {
+	config, _ := core.ReadConfigFiles(nil, nil)
+	if len(hashFunction) > 0 {
```

Comment:
is this really what you meant given this is a `string` (but `hashCheckers` is a `[]string`)?

Reply:
Funny enough I thought about that, but the end result should be the same (as opposed to `hashFunction != ""`). I guess, idiomatically speaking and for consistency `!= ""` is more in line with checking empty strings. Happy to change.